frontend/src/components/messstelle/optionsmenue/panels/TagesTypRadiogroup.vue

```diff
@@ -0,0 +1,85 @@
+<template>
+    <v-row no-gutters>
+        <v-col cols="6">
+            <v-radio-group
+                v-model="chosenOptionsCopy.tagesTyp"
+                class="full-width"
+            >
+                <v-radio
+                    :value="TagesTyp.WERKTAG_DI_MI_DO"
+                    :label="getTagesTypText(TagesTyp.WERKTAG_DI_MI_DO)"
+                />
+                <v-radio
+                    :value="TagesTyp.WERKTAG_MO_FR"
+                    :label="getTagesTypText(TagesTyp.WERKTAG_MO_FR)"
+                />
+                <v-radio
+                    :value="TagesTyp.SAMSTAG"
+                    :label="getTagesTypText(TagesTyp.SAMSTAG)"
+                />
+                <v-radio
+                    :value="TagesTyp.SONNTAG_FEIERTAG"
+                    :label="getTagesTypText(TagesTyp.SONNTAG_FEIERTAG)"
+                />
+                <v-radio
+                    :value="TagesTyp.WERKTAG_FERIEN"
+                    :label="getTagesTypText(TagesTyp.WERKTAG_FERIEN)"
+                />
+                <v-radio
+                    :value="TagesTyp.DTV"
+                    :label="getTagesTypText(TagesTyp.DTV)"
+                />
+            </v-radio-group>
+        </v-col>
+        <v-col cols="6"> {{ helperText }}</v-col>
```

Comment:
Die Seite wird immer in 3 Spalten der Breite 4 geteilt. Bitte anpassen, damit es einheitlich bleibt.